kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/GroupCoordinator.java

```diff
@@ -555,20 +558,24 @@ private void doSyncGroup(GroupMetadata group,
 
     public CompletableFuture<Errors> handleLeaveGroup(
         String groupId,
-        String memberId
+        Set<String> members
     ) {
         return validateGroupStatus(groupId, ApiKeys.LEAVE_GROUP).map(CompletableFuture::completedFuture
         ).orElseGet(() -> groupManager.getGroup(groupId).map(group -> group.inLock(() -> {
-            if (group.is(Dead) || !group.has(memberId)) {
+            if (group.is(Dead)) {
+                return CompletableFuture.completedFuture(Errors.UNKNOWN_MEMBER_ID);
+            } else if (!members.stream().allMatch((memberId) -> group.has(memberId))) {
```

Comment:
```suggestion
            } else if (!members.stream().allMatch(group::has)) {
```
Lambda can be replaced with method reference